go-controller/pkg/libovsdb/util/acl.go

```diff
@@ -127,8 +127,44 @@ func BuildACL(dbIDs *libovsdbops.DbObjectIDs, priority int, match, action string
 	return ACL
 }
 
+func BuildACL(dbIDs *libovsdbops.DbObjectIDs, priority int, match, action string, logLevels *ACLLoggingLevels,
```

Comment:
NIT: BuildDefaultTierACL should just call BuildACL with the default tier.
Also, missing doc block :)